scripts/scripts.js

```diff
@@ -209,6 +267,8 @@ async function loadLazy(doc) {
   const element = hash ? doc.getElementById(hash.substring(1)) : false;
   if (hash && element) element.scrollIntoView();
 
+  decorateSectionImages(doc);
```

Comment:
is it possible to use `createOptimisedPicture` from aem.js for this? @mkbansal1

Reply:
That method use single image only, we have 3 images, one for each viewport